src/vs/workbench/contrib/editSessions/browser/editSessionsViews.ts

```diff
@@ -47,7 +47,7 @@ export class EditSessionsDataViews extends Disposable {
 		const viewsRegistry = Registry.as<IViewsRegistry>(Extensions.ViewsRegistry);
 		viewsRegistry.registerViews([<ITreeViewDescriptor>{
 			id: viewId,
-			name,
+			name: { value: name, original: name },
```

Comment:
This one should be updated by either making `EDIT_SESSIONS_TITLE` an `ILocalizedString` or have this be:

```ts
name: { value: name, original: '......' },
```